plugins/@grouparoo/pardot/__tests__/.env.example

```diff
@@ -0,0 +1,5 @@
+# Note: These are only needed when recording new nock fixtures
+SALESFORCE_USERNAME = "demo@grouparoo.com"
+SALESFORCE_PASSWORD = "jhfd!kjdgd43h"
+SALESFORCE_SECURITY_TOKEN = "986273e87a3d9725fe3755c"
+PARDOT_BUSINESS_UNIT_ID = "0Uvabcd3fgh1jklmn0"
```

Comment:
it looks like there are some custom fields added. put that list here to setup env correctly. See other plugins like zendesk.

Reply:
I was debating whether I should just document this like I saw other plugins were doing it or create/delete the test fields when running the test since there's an endpoint for that. The latter may be more complex than is necessary though...

Reply:
the comments are fine